Fix devspace dev panic: -n shorthand conflicts with built-in --namespace flag

## Bug

`devspace dev` and `devspace dev --no-logs` both panic on startup:

```
panic: unable to redefine 'n' shorthand in "dev" flagset: it's already used for "no-logs" flag
```

## Root Cause

The `-n` short flag defined in the pipeline's `flags:` section conflicts with DevSpace's built-in `-n` / `--namespace` global flag.

## Fix

Remove the `short: n` line from the `no-logs` flag definition. The flag will only be accessible as `--no-logs` (no short form).

```yaml
flags:
  - name: no-logs
    description: "Disable container log streaming"
```

--- a/devspace.yaml
+++ b/devspace.yaml
@@ -7,7 +7,6 @@ pipelines:
   dev:
     flags:
       - name: no-logs
-        short: n
         description: "Disable container log streaming"
     run: |-
       if kubectl get application gateway -n argocd >/dev/null 2>&1; then

